app/services/host_fact_importer.rb

```diff
@@ -0,0 +1,56 @@
+class HostFactImporter
+  include Foreman::TelemetryHelper
+
+  attr_reader :host
+
+  def initialize(host)
+    @host = host
+  end
+
+  def create_new_record_when_facts_are_uploaded?
+    Setting[:create_new_host_when_facts_are_uploaded]
+  end
+
+  def skip_orchestration?
+    true
```

Comment:
do we actually need this method considering it's always true now? we could also simplify all the conditions using it

Reply:
Nah that's not needed right. Can you do another Ref commit @ezr-ondrej ?

Reply:
I wanted to keep it easy if someone decides to skip the orchestration in plugin eg, but I guess it just adds complexity for no reason :)

Reply:
Did this ever happen? Also, looks like there was no `fixes` commit so that the issue is still open

Reply:
Yeah, I did not follow through on this one.
The `skip_orchestration?` is kept, as lzap was too fast on merge.
I'll follow up with another PR.